main.cpp

```diff
@@ -52,85 +51,86 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: husky
+//  action 1: the husky pulls a sled
+husky.pullSled();
+//  action 2: the husky runs
+husky.run();
+//  action 3: the husky falls asleep
+husky.sleep();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: amplifier
+//  action 1: the amplifier's tone is adjusted
+amplifier.changeTone();
+//  action 2: the amplifier reads the signal input
+amplifier.readSignalInput();
+//  action 3: the amplifier boosts signal input
+amplifier.boostSignal();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: flower
+//  action 1: the flower ingrains it's roots into the ground
+flower.root();
+//  action 2: the flower grows
+flower.grow();
+//  action 3: the flower dies
+flower.die();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: water
+//  action 1: the water freezes
+water.freeze();
+//  action 2: the water leaks from it's container
+water.leak();
+//  action 3: the water drains
+water.drain();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: mouth
+//  action 1: they smiled 
+mouth.smile();
+//  action 2: they frowned 
+mouth.frown();
+//  action 3: they opened their mouth
+mouth.open();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: fan
+//  action 1: the fan blows air
+fan.blowAir();
+//  action 2: the fan blows harder
+fan.increasedAirFlow();
+//  action 3: the fan draws in air from outside
+fan.drawInAir();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: bird
+//  action 1: the bird takes flight
+bird.fly();
+//  action 2: the bird chirps
+bird.chirp();
+//  action 3: the bird laid an egg
+bird.layEgg();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.